Statistics/Distributions/LogPearson3.cs

```diff
@@ -54,9 +54,9 @@ public LogPearson3()
             SampleSize = 1;
             Min = double.NegativeInfinity;
             Max = double.PositiveInfinity;
+            _ProbabilityRange = IRangeFactory.Factory(0D,1D);
```

Comment:
@HenryGeorgist why do we need this if probability range is being set within BuildFromProperties?

Reply:
because build from properties uses the probability range in the cdf call when it trys to build probability range...

Reply:
Okay, then do we also need this to be in the constructor for a truncated distribution?

Reply:
so basically we set the probability range and then reset the probability range?

Reply:
i think the answer is yes - please test and verify 
create a test with a truncated distribution for lp3 - see if the test passes - if not add a default range of 0-1. and check for passing.

Reply:
R package is not working, it seems USGS stopped maintaining the package. I will attempt to use SSP.